docs/quickstart-eks.md

```diff
@@ -60,6 +60,22 @@ In this section, you will deploy an Amazon Elastic Container Service for Kuberne
                         --nodes=3 \
                         --version=${AWS_EKS_K8S_VERSION}
   ```
+  > **NOTE**: At the time of this writing, EKS clusters created with `eksctl` are affected by a [bug](https://github.com/awslabs/amazon-eks-ami/issues/193) that causes Elasticsearch to get into a crashloop. A temporary workaround consists of overriding the AMI used when creating the cluster. The AMI named `amazon-eks-node-1.10-v20190211` is known to work. You will need to find its ID that corresponds to the region and zone where you are creating the cluster. For instance:
```

Comment:
> You will need to find its ID

Can we include the relevant command that dumps out the below IDs table?

Reply:
Done.

Reply:
Done.